tests/service/data/SyntaxTree/Nullness/MatchWithTypeCastParensAndSeparateNullCase.fs

```diff
@@ -0,0 +1,2 @@
+match x with
+| :? (string | null) | null -> ()
```

Comment:
During typechecking of the matches, code like this could be warning - since the null part would be already covered by the first clause.